Fix 
$ make
gcc -g   -c -o si1132.o si1132.c
si1132.c:2:10: fatal error: wiringPiI2C.h: No such file or directory
    2 | #include <wiringPiI2C.h>
      |          ^~~~~~~~~~~~~~~

--- a/python_weather/BME280.py
+++ b/python_weather/BME280.py
@@ -14,6 +14,15 @@ power_mode = {
         "BME280_SOFT_RESET_CODE"    : 0xB6,
 }
 
+oversampl = {
+    "BME280_OVERSAMP_SKIPPED"       : 0x00,
+    "BME280_OVERSAMP_1X"            : 0x01,
+    "BME280_OVERSAMP_2X"            : 0x02,
+    "BME280_OVERSAMP_4X"            : 0x03,
+    "BME280_OVERSAMP_8X"            : 0x04,
+    "BME280_OVERSAMP_16X"           : 0x05,
+}
+
 regs = {
         "BME280_CHIP_ID_REG"            : 0xD0,
         "BME280_RST_REG"                : 0xE0,

--- a/python_weather/weather_board.py
+++ b/python_weather/weather_board.py
@@ -10,7 +10,11 @@ if len(sys.argv) != 2:
     sys.exit()
 
 si1132 = SI1132.SI1132(sys.argv[1])
-bme280 = BME280.BME280(sys.argv[1], 0x03, 0x02, 0x02, 0x02)
+bme280 = BME280.BME280(sys.argv[1],
+        BME280.power_mode['BME280_NORMAL_MODE'],
+        BME280.oversampl['BME280_OVERSAMP_1X'],
+        BME280.oversampl['BME280_OVERSAMP_1X'],
+        BME280.oversampl['BME280_OVERSAMP_1X'])
 
 def get_altitude(pressure, seaLevel):
     atmospheric = pressure / 100.0

